src/views/Project/index.jsx

```diff
@@ -22,15 +41,57 @@ const tagReposMapping = repositories =>
   }, {});
 
 @hot(module)
-@graphql(issuesQuery, {
-  skip: props => !projects[props.match.params.project].repositories,
-  options: () => ({
-    fetchPolicy: 'network-only',
-    variables: {
-      searchQuery: '',
-    },
+@compose(
+  graphql(issuesQuery, {
+    skip: ({
+      match: {
+        params: { project },
+      },
+    }) => !projects[project].repositories,
+    name: 'github',
+    options: () => ({
+      fetchPolicy: 'network-only',
+      variables: {
+        searchQuery: '',
+      },
+      context: {
+        client: 'github',
+      },
+    }),
   }),
-})
+  graphql(bugsQuery, {
+    skip: ({
+      match: {
+        params: { project },
+      },
+    }) => !projects[project].products,
+    name: 'bugzilla',
+    options: ({
+      match: {
+        params: { project },
+      },
+    }) => ({
+      variables: {
+        search: {
+          ...bugzillaSearchOptions,
+          products: productsWithNoComponents(projects[project].products).length
+            ? productsWithNoComponents(projects[project].products)
+            : Object.keys(projects[project].products[0])[0],
+          components: productsWithNoComponents(projects[project].products)
+            .length
+            ? undefined
+            : Object.values(projects[project].products[0])[0],
+        },
```

Comment:
A neater way than doing `components: null` would be not to provide the `components` at all:

```jsx
search: {
  ...bugzillaSearchOptions,
  ...(productsWithNoComponents(projects[project].products).length
    ? { products: productsWithNoComponents(projects[project].products) }
    : {
        products: Object.keys(projects[project].products[0])[0],
        components: Object.values(projects[project].products[0])[0],
      }),
},
```